### DISCUSSION TITLE
Plugin flags at app level

### DISCUSSION THREAD

**Discussion by jessecarterMOOSE** [idaholab/moose#21190] [Q&A Getting Started] [answered]
Is there anything that can be done at the app-level to change compile and/or include flags for plugins? I have an app with some UMAT  f90 files in a "plugins" directory on the app's root. I'm trying to add extra include dirs and compile flags to the Makefile but haven't managed to affect the way the plugins are being compiled.

@dschwen

**Comment by GiudGiud** [comment, marked as answer]
@dschwen

**Reply by GiudGiud** [reply]
@jessecarterMOOSE 
the easiest way to lure @dschwen to answer a question is to post the wrong answer haha

or maybe just reach out directly on slack

**Reply by jessecarterMOOSE** [reply]
LOL, noted. I think I figured it out though: set the `PLUGIN_FLAGS` variable in the app's Makefile. For example:

```
PLUGIN_FLAGS += -ffree-line-length-none -I$(CURDIR)/include
```
